apps/system/style/window.css

```diff
@@ -591,6 +591,18 @@ html[dir="rtl"] .appWindow.invoked {
   100% { transform: translateY(calc(-100%)); }
 }
 
+/*
+  When a mozbrowser iframe requests fullscreen with SHB enabled,
+  Gecko will make that element fixed and take up the entire viewport.
+  However, due to bug 879443 the transform property will make fixed
+  relative to this ancestor rather than the viewport, so we unset
+  the transform for this case.
+  See: https://bugzilla.mozilla.org/show_bug.cgi?id=1212960
+*/
+#screen.fullscreen-ancestor > #windows > .appWindow:not(.homescreen):not(.lockScreenWindow):not(.attentionWindow):not(.lockScreenInputWindow).active {
+  transform: unset;
+}
+
```

Comment:
In effect there is no difference, and that is what we are going for. According to [1], the -moz-full-screen-ancestor is going away, so I was proposing an alternative to that. I'm happy to use -moz-full-screen-ancestor since it's more consistent and we can then change everything over at once later.

1.) https://bugzilla.mozilla.org/show_bug.cgi?id=1212960#c11